### ISSUE TITLE
Update is not updating the application

### ISSUE THREAD

**Issue by arman-g** [microsoft/BotFramework-Emulator#2024]
The application does not update when clicked the Restart button.

## Version
Emulator v4.6

**Comment by anusharavindrar** [comment]
@arman-g could you elaborate more on what you are trying to do and the steps to reproduce(possibly with screenshots) ?

**Comment by arman-g** [comment]
@anusharavindrar the application is not updating when clicked "Restart" button. No error message is shown, the button simple does nothing. Have you guys tested this part before the release?

![emulator](https://user-images.githubusercontent.com/22778273/70932393-827b0800-1fee-11ea-83af-895985804ee0.PNG)

**Comment by anusharavindrar** [comment]
@arman-g I just checked for the same on my Windows work station and I am able to successfully update to v4.7.0 by clicking on the Restart option on the notifications tab. Could you mention the OS as well as the current emulator version you are using? Also, after closing and reopening the Emulator, does the issue still persist?

**Comment by cwhitten** [comment]
On Mac (Catalina) I was also able to update without issue.

**Comment by arman-g** [comment]
@anusharavindrar I am using windows 10 Enterprise on my work station and the Windows 10 Home addition on my personal desktop.
The current Emulator version is 4.6.0

**Comment by anusharavindrar** [comment]
@arman-g can you try to re-download the v4.6.0 from the Github release [page](https://github.com/microsoft/BotFramework-Emulator/releases/tag/v4.6.0) and then see if you are able to update to v4.7.0 after clicking on Restart from the notifications tab ?

**Comment by arman-g** [comment]
@anusharavindrar do you mean reinstall v4.6.0 again?

**Comment by anusharavindrar** [comment]
@arman-g yup. Can you try to download the v4.6.0 from the release pages on **one of your systems** and see if you are able to update to v4.7.0.

**Comment by arman-g** [comment]
@anusharavindrar Here are my tests.

1. I have emulator v4.6.0.
2. Downloaded v4.6.0 .exe from GitHub page you provided.
3. Reinstalled it without uninstalling it from my system (reinstalled it without any error)
4. The problem still persisted.

5. Uninstalled v4.6.0 from my system completely.
6. Installed v4.6.0 form previously downloaded exe.
7. I was able to update the application with "Restart" button.


I just uninstalled v4.6.0 from my stystem and installed it form github again. The button start working.
